lib/assignReviewers.js

```diff
@@ -0,0 +1,27 @@
+module.exports.assignReviewers = async function(context) {
+  pullRequestsPromiseObj = await context.github.pullRequests.getAll(
+    context.repo({per_page: 40}));
```

Comment:
Hmm, this is looking good but I see that a number of steps are common for the merge conflicts check and this check. Let's merge the two checks and put them in a single file. In that way, we will be able to reduce the number of requests made. What are your views?

Reply:
Yea that sounds better. I am on it

Reply:
Done